alacritty/src/config/ui_config.rs

```diff
@@ -48,6 +44,11 @@ pub struct UIConfig {
     /// Debug options.
     #[serde(default, deserialize_with = "failure_default")]
     pub debug: Debug,
+
+    // TODO - DEPRECATED
+    /// Use dynamic title.
```

Comment:
```suggestion
    // TODO: DEPRECATED
```

Follows what our deprecation comments look like everywhere else. Also, fairly certain we remove doc comments when we deprecate something.